common.blocks/menu/menu.js

```diff
@@ -134,8 +134,10 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : Control }, /** @lends menu.
     _onItemHover : function(item) {
         if(item.hasMod('hovered')) {
             this._hoveredItem && this._hoveredItem.delMod('hovered');
+            this.domElem.attr('aria-activedescendant', item.domElem.attr('id'));
```

Comment:
> Здесь достаточно динамически добавлять уникальный id при выборе пункта и удалять при снятии.
Лучше отказать от расстановки id в шаблонах и оставить всю логику в js на _onHoverItem.

I've spent some time thinking about this, and how I think this could work better. And even more: it seems, that you don't need to generate uniq id on every `hovered` call, as there can't be two `menu` (or `select` / `suggests`) with hovered items. All you  need is `identify(this)`